HybrasylXML/XSD/Castables.xsd

```diff
@@ -213,7 +213,7 @@
 
   <xs:complexType name="CastCostList">
     <xs:sequence>
-      <xs:element name="CastCost" type="actions:CastCost"/>
+      <xs:element name="CastCost" type="actions:CastCost"  minOccurs="1" maxOccurs="unbounded"/>
```

Comment:
CastCost should not have a minimum occurrance, as there as spells with no casting cost.

Reply:
Will fix, unintentional error.